braintree/credit-card-type
# Pull Request: Add Support for Egyptian Meeza Card

# Description
This pull request adds support for the Egyptian Meeza Card to the credit-card-type Node.js library, enabling developers to easily identify and validate Meeza Card numbers within their applications.

# Changes Made
- Included Meeza Card as a recognized card type in the credit-card-type library.
- Updated the documentation to include details on how to use the library to identify and validate Meeza Cards.

# How to Test
1. Clone this repository.
2. Install the library.
3. Use the library to check Meeza Card numbers in your Node.js application.
4. Ensure that Meeza Card numbers are correctly recognized and validated.

# Additional Information
- This update aligns the credit-card-type library with industry standards by recognizing the Meeza Card.
- It offers greater flexibility for developers in handling payments involving Meeza Cards.
- The changes are consistent with the library's coding style and maintain backward compatibility.

# Related Issues
I encountered difficulties in finding a library that supports Meeza Cards, which prompted me to contribute and add this support. This enhancement doesn't directly address a specific issue, but it aims to fill a significant gap in the library's card type recognition, benefiting developers dealing with Meeza Cards.
